### ISSUE TITLE
Validate schedule configurations for minLength/maxLength and pattern

### ISSUE THREAD

**Issue by benoit74** [openzim/zimfarm#835]
In the `/offliners` API, return minLength, maxLength and pattern.
Adapt offliner definitions (new field type to create) so that these values can be sourced from the definitions and automatically provided in the API.
In the UI, use these values to set them in the HTML5 elements and validate the input before submitting the form.
Also adapt zimit-frontend to use these values (https://github.com/openzim/zimit-frontend/issues/41).

**Comment by benoit74** [comment]
Removed from zimit2 project unfortunately

**Comment by elfkuzco** [comment]
@benoit74 , can you take a look at this? Seems to have been fixed as a consequence of #1246

**Comment by benoit74** [comment]
Yes, this is exactly #1246. To be deployed with `zimfarm-upgrade` branch.
